aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -472,10 +474,7 @@ impl<'r> WriteOpConverter<'r> {
                     },
                 }
             },
-            Some(existing_value) => match existing_value.into_metadata() {
-                None => WriteOp::Modification(data),
-                Some(metadata) => WriteOp::ModificationWithMetadata { data, metadata },
-            },
+            Some(_) => WriteOp::Modification(data),
```

Comment:
A few lines above we also have ` Some(metadata) => WriteOp::CreationWithMetadata { ...`. Why keeping this and not `Creation`? Although I think it doesn't really matter because aggregator did not distinguish `Create` vs `Modify` and when we embed it in the future this will be gone anyway

Reply:
Good point. I had to change the `ModificationWithMetadata` to `Modification` earlier when I ran sequential execution only with aggregators disabled. Now, we are enabling aggregators even for sequential execution. So, I reset the changes I made.